### PR TITLE
Polish measurement and signature verification

### PR DESCRIPTION
**Pull request by mssun**
## Description

Polish measurement and signature verification. Separate into two functions: `verify_enclave_info` and `load_enclave_info`. sdk and cli no longer need to verify signatures.

This is a perequiste of #158. The interfaces of sdk and cli have not been changed in this PR yet.

## Type of change (select applied and DELETE the others)

- [x] Code polishing

## How Has This Been Tested?
CI: https://ci.mesalock-linux.org/mssun/incubator-mesatee/140

## Checklist (check ALL before submitting PR, even not applicable)

- [x] Fork the repo and create your branch from `master`.
- [x] If you've added code that should be tested, add tests.
- [x] If you've changed APIs, update the documentation.
- [x] Ensure the tests pass (see CI results).
- [x] Make sure your code lints/format.

### CODE REVIEW

File: keys/auditors/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ sample entry looks like:
 ```
 kms
 
-mrsigner->value:
+mr_signer->value:
```
Comment (m4sterchain) [inline keys/auditors/README.md]: This is the original content of enclave.meta.txt, should be `mrsigner->value:`.

File: teaclave_utils/src/lib.rs [hunk 2]
```diff
@@ -73,16 +72,16 @@ pub fn percent_decode(orig: String) -> Result<String> {
...
+pub fn from_hex<'de, D>(deserializer: D) -> std::result::Result<SgxMeasurement, D::Error>
 where
     D: Deserializer<'de>,
 {
     use serde::de::Error;
     use serde::Deserialize;
     String::deserialize(deserializer).and_then(|string| {
         let v = decode_hex(&string).map_err(|_| Error::custom("ParseError"))?;
-        let mut array: SgxMeasure = [0; 32];
+        let mut array = [0; 32];
```
Comment (m4sterchain) [inline teaclave_utils/src/lib.rs]: use SGX_HASH_SIZE instead of 32.

File: mesatee_core/src/rpc/sgx/mod.rs [hunk 3]
```diff
@@ -60,40 +60,47 @@ pub fn prelude() {
...
+    }
+
+    let mut signatures: Vec<Vec<u8>> = vec![];
+    for ConfigSource::Path(ref path) in &RUNTIME_CONFIG.audit.auditor_signatures {
+        let signature =
+            fs::read(path).unwrap_or_else(|_| panic!("Cannot find signature file {:?}.", path));
+        signatures.push(signature);
+    }
+
+    for k in BUILD_CONFIG.auditor_public_keys {
```
Comment (m4sterchain) [inline mesatee_core/src/rpc/sgx/mod.rs]: I am expecting the verify_enclave_info taking the for-loop logic into its scope.

**Comment by mssun** [comment]
Updated.

**Pull request merged**